lib/DBM_Filter.pm

```diff
@@ -4,7 +4,8 @@ use strict;
 use warnings;
 our $VERSION = '0.06';
```

Comment:
You need to increment the version number.

Reply:
done